test/Quake/lambda_variable-2.qke

```diff
@@ -7,7 +7,8 @@
 // ========================================================================== //
 
 // RUN: cudaq-opt --lambda-lifting --canonicalize %s | FileCheck %s
-// RUN: cudaq-opt --lambda-lifting --canonicalize %s | cudaq-translate --convert-to=qir | FileCheck --check-prefixes=QIR %s
+// RUN: cudaq-opt --pass-pipeline='builtin.module(lambda-lifting,aggressive-early-inlining,func.func(lower-to-cfg),func.func(quake-add-deallocs))' %s \
```

Comment:
Minor note: I find it slightly easier to read command lines as a sequence of options as in the following.
```suggestion
// RUN: cudaq-opt --lambda-lifting --aggressive-early-inlining --lower-to-cfg --quake-add-deallocs %s \
```